draft-ietf-tokbind-https-08.xml

```diff
@@ -222,7 +222,7 @@
 
     <abstract>
       <t>This document describes a collection of mechanisms that allow
-      HTTP servers to cryptographically bind authentication tokens
+      HTTP servers to cryptographically bind authentication or authorization tokens
```

Comment:
I am fine with this if others are.  Though, in the rest of the abstract, we use the term "security tokens", so perhaps we ought to use it here rather than "authentication or authorization tokens" ?   

I suspect readers will generally realize that the parenthetical qualification of "(such as cookies and OAuth tokens)"  implies "authn or authz" tokens.